src/app/[locale]/list/_components-deprecated/AasList.tsx

```diff
@@ -0,0 +1,95 @@
+﻿import {
```

Comment:
you can ignore all files in "_components-deprecated" as I didn't change anything (those files will be removed after the new list is finished)